### DISCUSSION TITLE
Reading the length-constraint on a TextColumn

### DISCUSSION THREAD

**Discussion by StoppelJoshua** [simolus3/drift#2974] [Q&A] [answered]
Hey there,
I am looking for a way to read the length constraint of a table-column.
I created a table with a TextColumn inside:

```
class Mission extends Table {
   TextColumn get missionAlertByInput8 => text().withLength(max: 50)();
}
```

In my real-life application there are around 100 columns in that table and some of them have different LengthConstraints because of the backend-database.

To not manage all the changes with every TextField independently I created a custom TextForm-Widget that takes the column as an input and handles the data-writes and listens on the database based on that input column. All that is working perfectly. Now i would like to give that TextFormField a maxLength according to the input-column.
I know i can access the name of that column with ```Mission.missionAlertByInput8.$name``` which I am using to determine which streamProvider should be listened to and which column to update on a user-input. But is there any way to get the min/max-length?

thanks in advance

**Comment by simolus3** [comment, marked as answer]
Unfortunately, this information is not exposed in a way that would make it possible to efficiently query the min/max length. Drift just generates a validation function, but you won't get anything more than that callback.

**Reply by StoppelJoshua** [reply]
Alright. Thanks for the reply. I will have to handle that manually then.
